### ISSUE TITLE
Missing output file formats in mkvextract documentation

### ISSUE THREAD

**Issue by Alex131089** [mbunkus/mkvtoolnix#2094]
Hi, there's missing codec_id in output file formats of [mkvextract's documentation](https://mkvtoolnix.download/doc/mkvextract.html#mkvextract.output_file_formats) ; there's at least V_MPEGH/ISO/HEVC, S_TEXT/ASCII, S_TEXT/USF, S_HDMV/TEXTST, S_TEXT/WEBVTT,  missing.

I believe output formats are in https://github.com/mbunkus/mkvtoolnix/blob/release-15.0.0/src/extract/xtr_base.cpp#L139 with codec_id listed at https://github.com/mbunkus/mkvtoolnix/blob/release-15.0.0/src/common/codec.h#L27

With additional informations at:
* https://github.com/mbunkus/mkvtoolnix/blob/release-15.0.0/src/common/codec.cpp#L26
* https://github.com/mbunkus/mkvtoolnix/blob/release-15.0.0/src/common/file_types.cpp#L25

Thanks for mkvtoolnix :)

**Comment by Alex131089** [comment]
I got this missing : 
```
A_DTS
A_EAC3
A_MLP
A_PCM/INT/BIG
A_TRUEHD
S_HDMV/TEXTST
S_ASS
S_SSA
S_TEXT/ASCII
S_TEXT/WEBVTT
V_MPEG1
V_MPEG2
V_MPEGH/ISO/HEVC
```

For the story, I got this list by compiling codec list in https://gist.github.com/Alex131089/932271dfb106a9898ef232861100e3cf (if it can be useful) to make https://gist.github.com/Alex131089/98b1b8f6fce2d3365bfd14e44ff90580.

**Comment by remuxer32** [comment]
Fixed in 1c19827.
